autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,15 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers()
-    return ['go', 'golint']
```

Comment:
You can just add 'gometalinter' to the end of this list. Neomake checks that a maker exists before it tries to run it.